### PR TITLE
45 age verification

### PR DESCRIPTION
**Pull request by chhaack**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--sazerac-wheatleyvodka--hlxsites.hlx.page/
- After: https://45-age-verification--sazerac-wheatleyvodka--hlxsites.hlx.page/

### CODE REVIEW

**Comment by cziegeler** [comment]
The mobile view does not look right yet

File: scripts/scripts.js [hunk 1]
```diff
@@ -65,6 +100,7 @@ async function loadFonts() {
 function buildAutoBlocks(main) {
   try {
     buildHeroBlock(main);
+    buildAgeConfirmBlock(main);
```
Comment (karlpauls) [inline scripts/scripts.js:103]: We might consider adding the block to the LCP_BLOCKS

File: blocks/ageverification/ageverification.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+import { setCookie } from '../../scripts/scripts.js';
+/* age verification overlay */
+
+export default async function decorate(block) {
+  block.textContent = '';
+
+  const resp = await fetch('/templates/ageverification.plain.html', window.location.pathname.endsWith('/templates/ageverification') ? { cache: 'reload' } : {});
+
+  if (resp.ok) {
+    const html = await resp.text();
```
Comment (karlpauls) [inline blocks/ageverification/ageverification.js:11]: It seems like in the following you are reading out the information from the html and create new elements for all of it. Can't we work with the html we get from the template directly?

**Review by karlpauls — CHANGES_REQUESTED** [review]

**Comment by karlpauls** [comment]
@chhaack, a potentially interesting alternative would be to use a modal dialog. Not sure we need to but just in case you want to explore that direction there is an example in the block party: https://github.com/hannessolo/modal-demo/

File: blocks/ageverification/ageverification.js (hunk 2 above)
Comment (chhaack) [reply blocks/ageverification/ageverification.js:11]: i've tried to work with the html directly - i was not get it to work with css only. lots of manipulations in the dom are needed for making it work :-(.

**Comment by chhaack** [comment]
add "backdoor" ignoring age check cookie - use ?noAgeCheck as request parameter

File: blocks/ageverification/ageverification.js (hunk 2 above)
Comment (chhaack) [reply blocks/ageverification/ageverification.js:11]: new document layout (verification.docx) used - please check

**Review by karlpauls — CHANGES_REQUESTED** [review]
Looks pretty good overall now - however, if you go to a page that is not the root like e.g. https://45-age-verification--sazerac-wheatleyvodka--hlxsites.hlx.live/cocktails you'll see that some global styles are interfering and push a margin and max-width on you.

This needs to be fixed by either adding the block class to the not(.hero-container) selectors in the global styles or by overriding the settings.

**Review by karlpauls — APPROVED** [review]

**Pull request merged**